ui/src/test/workflowInsyde.test.js

```diff
@@ -0,0 +1,377 @@
+const timeout = 1000000;
+
+beforeAll(async () => {
+    await page.goto(URL, { waitUntil: "domcontentloaded" });
+    await page.setDefaultTimeout(0)
+    await page.setViewport({
+        width: 1920,
+        height: 1080
+    })
+});
+
+describe("Test upload", () => {
+
+   /**  test("Try upload without login", async () => {
+        const inputUploadHandle = await page.$('input[type=file]');
+
+        // prepare file to upload
+        let fileToUpload = 'C:/Users/nick1/Documents/o2r/o2r-UI-fork/ui/src/test/insyde_workspace.zip';
+        inputUploadHandle.uploadFile(fileToUpload);
+        await page.waitForTimeout(2000)
+        await page.click('#upload')
+        await page.waitForTimeout(2000)
+
+        const handle = await page.waitForSelector('#err', {visilbe: true})
+        const html = await page.evaluate(handle => handle.innerHTML, handle);
+
+        expect(html).toBe("This is the metadata we extracted out of your workspace. Is it correct? Fine, click the save button on the right. No? Make some changes and click on save.");
+    }, timeout);*/
+
+    test("Login", async () => {
+
+        let handle = await page.$("#login");
+        let html = await page.evaluate(handle => handle.innerText, handle);
+
+        if(html !== "LOGOUT"){
+            await page.click('#login')
+            //await page.waitForNavigation();
+            await page.waitForTimeout(3000)
+            await page.screenshot({ path: 'screenshots/insydelogin.jpg', type: 'jpeg' });
+            await page.click('#regular');
+            await page.waitForNavigation();
+        }
+
+        handle = await page.$("#login");
+        html = await page.evaluate(handle => handle.innerText, handle);
+
+        expect(html).toBe("LOGOUT");
+
+    }, timeout);
+
+    test("Try upload with login", async () => {
+
+        
+        const inputUploadHandle = await page.$('input[type=file]');
+        //await page.screenshot({ path: 'screenshots/StartPageLoggedIn.jpg', type: 'jpeg' });
+
+        // prepare file to upload
+        let fileToUpload = './src/test/insyde_workspace.zip';
+        inputUploadHandle.uploadFile(fileToUpload);
+        await page.waitForTimeout(2000)
+        await page.click('#upload')
+        await page.waitForNavigation({waitUntil: 'networkidle2'});
+        await page.waitForTimeout(2000)
+        //await page.waitForSelector("h3")
```

Comment:
Please remove commented out lines, or add a comment why these variants should be kept.